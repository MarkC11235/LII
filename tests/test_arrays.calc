x = 5
array a[10] = {0} # array of 10 elements, all initialized to 0
array b[10] = {0} # array of 10 elements, all initialized to 0
array c[2] = {63, 42} # array of 2 elements, initialized to 63 and 42

for(i = 0; i < 10; i = i + 1){
    array a[i] = 9 - i # assign to array a[i] the value 9 - i
    array b[i] = i # assign to array b[i] the value i
}

for(i = 0; i < 10; i = i + 1){
    a[i] + b[i] # 9 + 0, 8 + 1, 7 + 2, 6 + 3, 5 + 4, 4 + 5, 3 + 6, 2 + 7, 1 + 8, 0 + 9
}

c[0] # 63
c[1] # 42