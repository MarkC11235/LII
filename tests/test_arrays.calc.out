9
9
9
9
9
9
9
9
9
9
63
42